server/protocols/webadmin/webadmin-core/src/main/java/org/apache/james/webadmin/routes/HealthCheckRoutes.java

```diff
@@ -80,10 +82,10 @@ public void define(Service service) {
     public void validateHealthchecks() {
         service.get(HEALTHCHECK,
             (request, response) -> {
-                List<Pair<ComponentName, Result>> anyUnhealthy = retrieveUnhealthyHealthChecks();
+                List<Pair<ComponentName, Result>> anyUnhealthyOrDegraded = retrieveUnhealthyOrDegradedHealthChecks();
```

Comment:
As there's no content in the response yet, I prefer to answer 500 when a check is degrade.
This way the user will have to check the logs in order to know what's happening.